Content.Client/GameObjects/Components/AcceptCloningWindow.cs

```diff
@@ -0,0 +1,57 @@
+﻿#nullable enable
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Shared.Localization;
+
+namespace Content.Client.GameObjects.Components
+{
+    public sealed class AcceptCloningWindow : SS14Window
+    {
+        public readonly Button DenyButton;
+        public readonly Button ConfirmButton;
+
+        public AcceptCloningWindow(ILocalizationManager loc)
+        {
+            var localization = loc;
```

Comment:
I thought we're purging ILocalizationManager and using the static `Loc.GetString` instead?